src/components/UpgradePlanForm/UpgradePlanForm.js

```diff
@@ -30,98 +31,104 @@ class UpgradePlanForm extends React.Component {
     this.setState({ availablePlans: ClientTypePlans });
   }
 
-  async handleSubmit(event) {
-    event.preventDefault();
+  async onSubmit(values, { setSubmitting }) {
```

Comment:
Finally, I think that _onSubmit_ is more clear than _handleSubmit_ (because it is not handling the HTML form submit directly)

Thoughts?

Reply:
Crystal clear